src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -59,7 +59,7 @@ public EditCommand parse(String args) throws ParseException {
 
         parseGroupTagsForEdit(argMultimap.getAllValues(Prefix.GROUP_TAG))
```

Comment:
Same here, perhaps better for parse functions to be in ```ParserUtil```

Reply:
I moved the other parse function into util already but yea maybe can move this one too